src/routes/Landing.tsx

```diff
@@ -45,9 +46,10 @@ const getWidgetLayoutLandingPage = () => {
 };
 
 const Landing = () => {
-  const widgetLayoutLandingPageEnabled = useFlag(
-    'platform.landing-page.widgetization'
-  );
+  const { isProd, isBeta } = useChrome();
+  const widgetLayoutLandingPageEnabled =
+    useFlag('platform.landing-page.widgetization') &&
+    (!isProd() || (isProd() && isBeta())); // TODO remove once prod-stable release ready
```

Comment:
Don you just want to create an extra flag from prod preview/stable and allow the widgets if at least one is true?

Reply:
That wouldn't work. We have one instance of unleas for production, enabling either of those flags would enable it for both preview and stable.

However I can create two FFs and check if stable environment and disable based on that. That way we don't have to touch code and just enable it!